supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -27,6 +28,15 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
 
   private implicit val ec = context.dispatcher
   private val maxRetries = 10
+  // FIXME: Determined empirically that we start to get errors when the batch request approaches 15MB.
```

Comment:
What would be interesting is to monitor the size (both as in number of requests and total byte size) per JES batch. It would give us an idea of the limiting factor.
Another way to do this would be to not limit the byte size of the request but simply cut it in half if it fails until it succeeds...
Also it looks like the limit for number of queries is actually 1000 even though we limit ourselves to 100 https://cloud.google.com/genomics/batch